### ISSUE TITLE
Convert paired-end RNA-seq to single-read RNA-seq?

### ISSUE THREAD

**Issue by junjunlab** [YangLab/CIRCexplorer2#69]
Hi,
Thanks for this great tool. The **CIRCexplorer2 align** only supports single-end read data. The documentation mentioned **It is recommended to convert paired-end RNA-seq to single-read RNA-seq before alignment.** Can I **zcat test1.fq.gz test2.fq.gz > test.fq.gz** as input? Thanks!

best wished,
JunZhang

**Comment by kepbod** [comment]
Yes, you could do this. If you want to use paired-end reads directly, I recommend you to use the STAR pipeline.

**Comment by junjunlab** [comment]
Hi,
I want to use **fast_circ.py annotate** one command to run. But I got an error **buitie2-build: unrecognized option '--threads'**. I think the reason is that my python version is 2.7. **CIRCexplorer2** can support python3 after version 2.3.0. But tophat2 only support python2.7 which means I could not use **fast_circ.py annotate** to run this pipeline in python2/3? Thanks!

best wished,
JunZhang

**Comment by kepbod** [comment]
Could you offer the full information of errors?
